doc/lsst.meas.base/index.rst

```diff
@@ -17,15 +17,55 @@ Using lsst.meas.base
    :maxdepth: 1
 
    intro
-   reconstructing-measurements
+   tasks_and_algorithms
+   reconstructing_measurements
 
 Contributing
 ============
 
 ``lsst.meas.base`` is developed at https://github.com/lsst/meas_base.
 You can find Jira issues for this module under the `meas_base <https://jira.lsstcorp.org/issues/?jql=project%20%3D%20DM%20AND%20component%20%3D%20meas_base>`_ component.
 
+Task reference
+==============
+
+Command-line tasks
+------------------
+
+.. lsst-cmdlinetasks::
+   :root: lsst.meas.base
+
+Tasks
+-----
+
+.. lsst-tasks::
+   :root: lsst.meas.base
+   :toctree: tasks
+
+Configs
+-------
+
+.. lsst-configs::
+   :root: lsst.meas.base
+   :toctree: configs
+
 Python API reference
 ====================
 
-.. .. automodapi:: lsst.meas.base
+.. There are two reasons for the :skip: parameters below.
+   - Objects which are imported with lsst.meas.base but aren't actually part
+     of it --- e.g. lsst.meas.base.Calib is actually lsst.afw.image.calib.Calib
+     --- cause the Sphinx build to fail when executed standalone.
+   - SingleFrameFromGenericPlugin and ForcedFromGenericPlugin cause the
+     construction of an inheritance diagram to fail, per DM-15461.
+     Unfortunately, including them here causes a warning, but that seems to be
+     unavoidable.
```

Comment:
Thank you for the! That said I am really surprised about item 1. I have never seen such a failure and the packages I work on have symbols hoisted into the main namespace. I prefer the documentation to appear there.

Reply:
I also found this peculiar. The failure I get if I remove these lines is of the form:
```
$ package-docs clean && package-docs build
 [ elided ]
Traceback (most recent call last):
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/documenteer/sphinxrunner.py", line 85, in run_sphinx                                                                                  
    app.build(force_all, filenames)
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/application.py", line 325, in build                                                                                            
    self.builder.build_all()
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/builders/__init__.py", line 299, in build_all                                                                                  
    self.build(None, summary='all source files', method='all')
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/builders/__init__.py", line 359, in build                                                                                      
    for docname in self.env.check_dependents(self.app, updated_docnames):
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/__init__.py", line 623, in check_dependents                                                                        
    for docnames in app.emit('env-get-updated', self):
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/application.py", line 444, in emit                                                                                             
    return self.events.emit(event, self, *args)
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/events.py", line 79, in emit
    results.append(callback(*args))
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/collectors/toctree.py", line 135, in get_updated_docs                                                              
    return self.assign_section_numbers(env) + self.assign_figure_numbers(env)
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/collectors/toctree.py", line 278, in assign_figure_numbers                                                         
    _walk_doc(env.config.master_doc, tuple())
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/collectors/toctree.py", line 275, in _walk_doc                                                                     
    _walk_doctree(docname, doctree, secnum)
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/collectors/toctree.py", line 253, in _walk_doctree                                                                 
    _walk_doctree(docname, subnode, secnum)
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/collectors/toctree.py", line 269, in _walk_doctree                                                                 
    _walk_doctree(docname, subnode, secnum)
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/collectors/toctree.py", line 261, in _walk_doctree                                                                 
    _walk_doc(subdocname, secnum)
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/collectors/toctree.py", line 275, in _walk_doc                                                                     
    _walk_doctree(docname, doctree, secnum)
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/collectors/toctree.py", line 253, in _walk_doctree                                                                 
    _walk_doctree(docname, subnode, secnum)
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/collectors/toctree.py", line 253, in _walk_doctree                                                                 
    _walk_doctree(docname, subnode, secnum)
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/collectors/toctree.py", line 253, in _walk_doctree                                                                 
    _walk_doctree(docname, subnode, secnum)
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/collectors/toctree.py", line 269, in _walk_doctree                                                                 
    _walk_doctree(docname, subnode, secnum)
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/collectors/toctree.py", line 261, in _walk_doctree                                                                 
    _walk_doc(subdocname, secnum)
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/collectors/toctree.py", line 274, in _walk_doc                                                                     
    doctree = env.get_doctree(docname)
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/__init__.py", line 785, in get_doctree                                                                             
    with open(doctree_filename, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/jds/Projects/LSST/src/meas_base/doc/_build/doctree/py-api/lsst.meas.base.Calib.doctree'                                                    

Exception occurred:
  File "/Users/jds/sw/venv/numpydoc-venv/lib/python3.6/site-packages/sphinx/environment/__init__.py", line 785, in get_doctree                                                                             
    with open(doctree_filename, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/jds/Projects/LSST/src/meas_base/doc/_build/doctree/py-api/lsst.meas.base.Calib.doctree'                                                    
The full traceback has been saved in /var/folders/g9/5c5t1myd54scz1kz39g718b40000gn/T/sphinx-err-s5fjgu1k.log, if you want to report the issue to the developers.                                          
Please also report this if it was a user error, so that a better error message can be provided next time.                                                                                                  
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```
I don't think figuring out what's going on is in scope for this ticket, but if @jonathansick has any ideas I'd be very interested to hear them!